src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,24 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val result: MutableList<String> = mutableListOf()
+        val greeting: HelloWorldGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world, Salut tout le monde, Hola, mundo"
+            }
+
+            override fun greetSomeone(someone: String): String {
+
+                return "Hello $someone, Salut $someone, Hola, $someone"
+            }
+        }
+
+        result.addAll(greeting.greet().split(", ", limit = 3))
```

Comment:
Try to find more flexible solution.
Think if there are 100 greeting items or 1000